releasenotes/notes/holland-vevn-9eb62140a2c8ff56.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - Holland now runs inside a python virtual environment
+    by default. By default, the virtual environment will
+    be placed in /openstack/venvs/holland-release-tag.
+upgrade:
+  - On upgrades, the holland pip packages will linger in 
+    the global environment if not cleaned up.
```

Comment:
actually sorry, I understand this more now. I still think we need a note about what should be done to clean this up. Or potentially we need to add a card to address this lingering stuff in the upgrade work 

/cc @git-harry